### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by bfreskura**

### CODE REVIEW

File: jeredocker_mmagic/Dockerfile
Comment (jere357) [inline jeredocker_mmagic/Dockerfile:1]: i think we should keep this mmagic Dockerfile bcs it had some weird tricks that were needed in order for openmmlab code to wrok

**Comment by jere357** [comment]
ok @bfreskura we gotta consistently name the groups, on train2 the group is aimages on karijola the group is company, choose one and make it the same for all machines

File: jeredocker_mmagic/Dockerfile
Comment (bfreskura) [reply jeredocker_mmagic/Dockerfile:1]: But we aren't using it. When and if we start using it, the installation steps may become outdated. 

That's why I wouldn't include it. We can always take a peek in the commit history if need to.

**Comment by bfreskura** [comment]
> ok @bfreskura we gotta consistently name the groups, on train2 the group is aimages on karijola the group is company, choose one and make it the same for all machines EDIT: this wasnt the issue but ok

It's up to the user to setup the container with proper groups.

**Review by jere357 — APPROVED** [review]

**Pull request merged**